### PR TITLE
Set up pre-commit linting hooks

### PR DESCRIPTION
**Pull request by illright**
This still will not add workspaces. Also I hope workspaces aren't exclusive to Yarn 2. We are using Yarn 1, and I tried to move to Yarn 2, but it was such a mess, Yarn 1 seemed a lot friendlier than whatever they turned it into, so I don't want to make the switch.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
-/node_modules/
+/node_modules
```
Comment (aabounegm) [inline .eslintignore:1]: How about any `node_modules` in general? (Remove the slash)

File: .stylelintignore [hunk 2]
```diff
@@ -1,5 +1,8 @@
 node_modules/
 demo/public/
+demo/node_modules/
```
Comment (aabounegm) [inline .stylelintignore:3]: Same here. Just `node_modules` and `__sapper__`. I don't think there is ever a case where we might need to include one of them

**Comment by aabounegm** [comment]
Workspaces are available since Yarn v1.0. Do you want me to add them here or in a different PR (if we decide to use them)?

**Comment by illright** [comment]
Preferably a different PR

**Pull request merged**